python/lsst/dax/obscore/obscore_exporter.py

```diff
@@ -135,7 +141,18 @@ def __init__(self, fname: str, null_value: bytes, sep_in: bytes, sep_out: bytes)
         self.buffer: bytes = b""
 
     def write(self, buffer: bytes) -> int:  # type: ignore
-        """Write next buffer to output."""
+        """Write next buffer to output.
+
+        Parameters
+        ----------
+        buffer : `bytes`
+            Buffer to write.
+
+        Returns
+        -------
+        `int`
```

Comment:
```suggestion
        size: `int`
```

Reply:
Ironically numpydoc itself prefers not to have a variable name if there is only one thing being returned. It's the Rubin style guide that insists on the name. Fixed.